jupyterhub/templates/_helpers.tpl

```diff
@@ -4,6 +4,41 @@
   objects we define in the .yaml template files.
 
 
+  ## How helpers work
+  Helm helper functions is a good way to avoid repeating something. They will
+  generate some output based on one single dictionary of input that we call the
+  helpers scope.
+
+  Often the current scope (.) is passed to the helper as it give the helper
+  access to the current scope containing .Release.Name, .Values.rbac.enabled and
+  similar values.
+
+  {{ include "jupyterhub.commonLabels" . }}
```

Comment:
This is an example of calling the helper called `jupyterhub.commonLabels` with scope `.`?